apps/E2E/src/common/BasePage.ts

```diff
@@ -177,25 +241,12 @@ export abstract class BasePage {
     }
   }
 
-  /* Waits for the test page to load. If the test page doesn't load before the timeout, it causes the test to fail. */
-  async waitForPageDisplayed(timeout?: number): Promise<void> {
-    await browser.waitUntil(async () => await this.isPageLoaded(), {
-      timeout: timeout ?? this.waitForUiEvent,
-      timeoutMsg: this._pageName + ' did not render correctly. Please see /errorShots for more information.',
-      interval: 1500,
-    });
+  /** Waits for the tester app to load by checking if the startup page loads. If the app doesn't load before the timeout, it causes the test to fail. */
+  async waitForInitialPageToDisplay(): Promise<void> {
+    await this.waitForCondition(async () => await (await this._initialPage).isDisplayed(), this.ERRORMESSAGE_APPLOAD, BOOT_APP_TIMEOUT);
   }
 
-  /* Waits for the test page's button to be displayed. If the button doesn't load before the timeout, it causes the test to fail. */
-  async waitForButtonDisplayed(timeout?: number): Promise<void> {
-    await browser.waitUntil(async () => await this.isButtonInView(), {
-      timeout: timeout ?? this.waitForUiEvent,
-      timeoutMsg: 'Could not find the button to navigate to ' + this._pageName + '. Please see /errorShots for more information.',
-      interval: 1500,
-    });
-  }
-
-  /* @deprecated, only use `scrollToTestElement()` instead */
+  /** @deprecated, only use `scrollToTestElement()` instead */
```

Comment:
Do we really need this here? Are there any uses of waitForPrimaryElementDisplayed() currently?

Since this isn't an external package, I don't think we really need to use deprecated here. We should just replace all instances of this with the new method and delete it